script.js

```diff
@@ -0,0 +1,49 @@
+const addBook = document.querySelector('#btn');
+const bookList = document.querySelector('#list');
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const form = document.querySelector('#form');
+const Book = function objBook(title, author) {
+  this.title = title;
+  this.author = author;
+};
+const storedBooks = [];
+function addBooks(newBook) {
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly use ES6 syntax for all your functions and methods as this is the recommended practice.